packages/hash/frontend/src/pages/[account-slug]/types/entity-type/[entity-type-id].page.tsx

```diff
@@ -61,13 +63,20 @@ const Page: NextPageWithLayout = () => {
     baseEntityTypeUri,
     (fetchedEntityType) => {
       reset({
-        properties: Object.values(fetchedEntityType.properties).map((ref) => {
-          if ("type" in ref) {
-            // @todo handle property arrays
-            throw new Error("handle property arrays");
-          }
-          return { $id: mustBeVersionedUri(ref.$ref) };
-        }),
+        properties: Object.keys(fetchedEntityType.properties).map(
+          (propertyId) => {
+            const ref = fetchedEntityType.properties[propertyId]!;
```

Comment:
This removes need to use `!` assertion 

```suggestion
        properties: Object.entries(fetchedEntityType.properties).map(
          ([propertyId, ref]) => {
```

Reply:
Thank you, that looks cleaner. 3ee46210d31a2f3ab358b62339b199762805e821